docs/general/testing/web/index.md

```diff
@@ -0,0 +1,25 @@
+---
+uid: testing-web-clients
+title: Testing Jellyfin Web Clients
+---
+
+## Weekly Unstable Builds
+
+Weekly unstable builds ship Jellyfin Server and Jellyfin Web in a complete package. Please refer to [Testing Jellyfin Server](/docs/general/testing/server/) for more info.
+
+## Testing PRs from Cloudflare Pages Deployments
+
+Each PR of [Jellyfin Web](https://github.com/jellyfin/jellyfin-web) and [Jellyfin Vue](https://github.com/jellyfin/jellyfin-vue) is automatically deployed to Cloudflare Pages. Please follow the steps below on how to test them.
+
+1. Find the PR you would like to checkout in the list of open PRs: [PRs for Jellyfin Web](https://github.com/jellyfin/jellyfin-web/pulls), [PRs for Jellyfin Vue](https://github.com/jellyfin/jellyfin-vue/pulls)
```

Comment:
>checkout

In this case it should be "check out" as in "look at, view, inspect"; "checkout" is a Git action. Though I'd use something like "PR you would like to demo in the" or "PR you would like to test in the" personally as it's less informal.